bench/wtperf/wtperf.c

```diff
@@ -1411,21 +1411,17 @@ flush_tier_worker(void *arg)
             break;
         /*
          * In order to get all the data into the last object when the work is done, we need to call
-         * checkpoint to get all the data into this object before calling flush_tier.
+         * checkpoint with flush_tier enabled to get all the data into this object.
```

Comment:
I'd leave off "to get all the data into this object", it's already in the sentence earlier.